js/components/app.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
try adding this line here. Should fix a few errors.

 `/* global LoginScreen, HomeScreen, NotFoundScreen */`